tensortrade/environments/trading_environment.py

```diff
@@ -47,14 +49,21 @@ def __init__(self,
                  reward_scheme: Union[RewardScheme, str],
                  feed: DataFeed = None,
                  window_size: int = 1,
-                 use_internal=True,
+                 use_internal: bool = True,
+                 render_mode: str = 'human',
+                 chart_height: int = 800,
```

Comment:
`chart_height` and `price_history` should be `kwargs`.

Reply:
Done in daceea2.